qutebrowser/config/configdata.yml

```diff
@@ -140,7 +140,7 @@ qt.force_software_rendering:
   default: false
   backend: QtWebEngine
   desc: >-
-    Force software rendering for QtWebEngine.
+    Whether to force software rendering for QtWebEngine.
```

Comment:
I'd like to keep the descriptions in [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood) if possible, without adding "Whether to". They just add unnecessary fluff here IMHO, which makes things slightly harder to read, and takes away space in the completion.